lib/vrf.c

```diff
@@ -1052,6 +1058,7 @@ int vrf_bind(vrf_id_t vrf_id, int fd, const char *ifname)
 		ifname = vrf->name;
 	}
 
+	zlog_debug("Bind to device for fd: %d %s", fd, ifname);
```

Comment:
Suggest to change it to.. 

zlog_debug("Bind to device for fd: %d interface %s", fd, ifname);